dashboard/config/blocks/Outbreak/Outbreak_beginCongregating.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
for my own understanding, when you create new blocks that need to be in a block pool, do you create them through the levelbuilder UI? how do you create a new pool?

Reply:
All blocks have a pool, and all the pools are listed here: `https://levelbuilder-studio.code.org/pools/`
To create a new pool, you just go to the URL `https://levelbuilder-studio.code.org/pools/<new pool name>/blocks/new` (which is a little hacky / not documented anywhere 😬)